build/bazel/remote/execution/v0.2/remote_execution.proto

```diff
@@ -994,6 +1009,78 @@ message GetTreeResponse {
   string next_page_token = 2;
 }
 
+// A request message for
+// [Capabilities.GetCapabilities][google.devtools.remoteexecution.v1test.Capabilities.GetCapabilities].
+message GetCapabilitiesRequest {
+  // The instance of the execution system to operate against. A server may
+  // support multiple instances of the execution system (with their own workers,
+  // storage, caches, etc.). The server MAY require use of this field to select
+  // between them in an implementation-defined fashion, otherwise it can be
+  // omitted.
+  string instance_name = 1;
+}
+
+// A response message for
+// [Capabilities.GetCapabilities][google.devtools.remoteexecution.v1test.Capabilities.GetCapabilities].
+message GetCapabilitiesResponse {
+  // Capabilities of the remote cache system.
+  CacheCapabilities cache_capabilities = 1;
+
+  // Capabilities of the remote execution system.
+  ExecutionCapabilities execution_capabilities = 2;
+}
+
+// The digest function used for converting values into keys for CAS and Action
+// Cache.
+enum DigestFunction {
+  UNKNOWN = 0;
+  SHA256 = 1;
```

Comment:
nit: Should these be SHA_256 and SHA_1 for style?

Reply:
I don't think so, I've seen this style used with hash function names much more often than not (results may be biased as they are based primarily on internal code searches :-)).